server/app/views/components/CreateQuestionButton.java

```diff
@@ -35,7 +35,7 @@ public static DivTag renderCreateQuestionButton(String questionCreateRedirectUrl
                 Styles.BG_WHITE,
                 Styles.TEXT_GRAY_600,
                 Styles.SHADOW_LG,
-                Styles.ABSOLUTE,
```

Comment:
I *believe* that the "absolute" class is required for positioning the dropdown appropriately next to the content.

I think the source of the problem is actually that we set 'overflow-hidden' in `QuestionBank.questionBankPanel`. Do you mind restoring the "absolute" tag here and removing the "overflow-hidden" tag in `QuestionBank.questionBankPanel`? If that works, there should be no need to set "overflow-auto".

Reply:
that works too!